### PR TITLE
rework topic details statistics bar

### PR DESCRIPTION
**Pull request by rikimaru0345**
remove dynamic config entries, just show retention/segment ms/bytes based on cleanup.policy

### CODE REVIEW

**Comment by rikimaru0345** [comment]
Changes:

1) no more dynamic (user defined) config properties; what config entries will be shown depends on cleanup policy
2) old table-style config display has been removed

image

File: frontend/src/components/pages/topics/QuickInfo.tsx [hunk 1]
```diff
@@ -42,19 +40,30 @@ export const TopicQuickInfoStatistic = observer((p: { topic: Topic }) => {
...
+
+        if (cleanupPolicy.includes('delete')) {
+            dynamicEntries.push('retention.ms');
+            dynamicEntries.push('retention.bytes');
+        }
+        if (cleanupPolicy.includes('compact')) {
+            dynamicEntries.push('segment.ms');
+            dynamicEntries.push('segment.bytes');
+        }
+
```
Comment (sago2k8) [inline frontend/src/components/pages/topics/QuickInfo.tsx:57]: [sand]: I see this and I notice we could reuse this logic in the future, not blocking at all, it's very easy to read and understand, but In my mind a helper function would be ideal, so you could potentially reuse it.

**Review by sago2k8 — APPROVED** [review]
LGTM, small comment not blocking

File: frontend/src/components/pages/topics/QuickInfo.tsx (hunk 1 above)
Comment (rikimaru0345) [reply frontend/src/components/pages/topics/QuickInfo.tsx:57]: Where would we reuse it?
What would the helper function do? 

Are you thinking about something like extracting name+value pairs of the config?
Or the logic where it switches what kinds of configs to display depending on cleanupPolicy?

**Pull request merged**